.github/workflows/branch-name-policy.yml

```diff
@@ -2,18 +2,18 @@ name: Fail when incorrect branch names are used
 on:
   push:
     branches:
-    - '*/*'
-    - '*/*/*'
+    - '*/**'
 
 jobs:
   no_slash_in_branch_names:
     runs-on: ubuntu-latest
     steps:
       - name: If this is triggered, it should fail
         run: |
-          echo "::error::This branch ${GITHUB_REF_NAME} has a slash; please don't use slashes in branch names"
-          echo "::error::Please close this PR and rename the local branch with"
-          echo "::error::    git checkout ${GITHUB_REF_NAME}"
-          echo "::error::    git branch -m branch-no-slashes"
+          __msg="::error::This branch ${GITHUB_REF_NAME} has a slash; please don't use slashes in branch names
+          Please close this PR and rename the local branch with
+              git checkout ${GITHUB_REF_NAME}
+              git branch -m ${GITHUB_REF_NAME//\//-}"
```

Comment:
Haha, nice :D Thought about suggesting something like this, but figured it's too much effort for a little message. Definitely prefer it like this tho!